src/core/server/http/cookie_session_storage.ts

```diff
@@ -31,11 +32,32 @@ export interface SessionStorageCookieOptions<T> {
 }
 
 class ScopedCookieSessionStorage<T extends Record<string, any>> implements SessionStorage<T> {
-  constructor(private readonly server: Server, private readonly request: Request) {}
+  constructor(
+    private readonly log: Logger,
+    private readonly server: Server,
+    private readonly request: Request
+  ) {}
   public async get(): Promise<T | null> {
     try {
-      return await this.server.auth.test('security-cookie', this.request as Request);
+      const session = await this.server.auth.test('security-cookie', this.request);
```

Comment:
Should this string be pulled from a shared constant?

Reply:
Don't think so.
1. Strategy name is declared in the same file: https://github.com/elastic/kibana/blob/5846fe460143611037957df04bd4abcc8d560411/src/core/server/http/cookie_session_storage.ts#L87
This knowledge shouldn't leak outside.
1. I believe that `shared constant` won't exist in the New platform. They are always a part of a domain and should be passed accordingly, with declaring direct dependencies.